tests/unit_tests/test_aws.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
+
+import pytest
+
+from sky.clouds.aws import AWS
+
+
+def test_aws_label():
+    aws = AWS()
+    # Invalid - AWS prefix
+    assert not aws.is_label_valid('aws:whatever', 'value')[0]
+    # Valid - valid prefix
+    assert aws.is_label_valid('any:whatever', 'value')[0]
+    # Invalid - Too long
+    assert not (aws.is_label_valid(
+        'sprinto:thisiexample_string_with_123_characters_length_thing_thing_thing_thing_thing_thing_thing_thin_thing_thing_thing_thing_thing_thing',
+        'value',
+    )[0])
+    # Invalid - Too long
+    assert not (aws.is_label_valid(
+        'sprinto:short',
+        'thisiexample_string_with_123_characters_length_thing_thing_thing_thing_thing_thing_thing_thin_thing_thing_thing_thing_thing_thingthisiexample_string_with_123_characters_length_thing_thing_thing_thing_thing_thing_thing_thin_thing_thing_thing_thing_thing_thing',
+    )[0])
```

Comment:
Let's inlcude the following test as well, which is widely used by our users ; )
```suggestion
    )[0])
    
    # Valid
    assert aws.is_label_valid('Owner', 'username-1')[0]
```

Reply:
Done